clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1010,48 +1010,109 @@ architecture to achieve optimal efficiency.  Google is sharing
 instruction and memory address traces from workloads running in Google
 data centers so that computer architecture researchers can study and
 develop new architecture ideas to improve the performance and
-efficiency of this important class of workloads.
+efficiency of this important class of workloads. To protect Google's
+intellectual property, these traces have been modified in order to
+filter out sensitive information. These traces follow a synthetic ISA
```

Comment:
Let's make this even more clear: sthg like "These traces have had their original ISA replaced with a synthetic ISA..."  (Otherwise it kind of sounds like we compiled and ran them on some weird ISA machine.)